flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskManagerTest.java

```diff
@@ -486,7 +489,8 @@ protected void run() {
 		public void onReceive(Object message) throws Exception {
 			if(message instanceof RegistrationMessages.RegisterTaskManager){
 				final InstanceID iid = new InstanceID();
-				getSender().tell(new RegistrationMessages.AcknowledgeRegistration(iid, -1),
+				getSender().tell(new RegistrationMessages.AcknowledgeRegistration(iid, -1,
+								Option.<ActorRef>apply(null)),
```

Comment:
Just curious, why not just pass None instead of {{Option.<ActorRef>apply(null)}} ?